### PR TITLE
Add too-many-lists repo

### PR DESCRIPTION
**Pull request by edunham**
Signed-off-by: E. Dunham <bonnycisignoff@edunham.net>

### CODE REVIEW

File: roles/zuul/templates/etc/zuul/config/layout.yaml [hunk 1]
```diff
@@ -104,3 +104,9 @@ projects:
       - tox_py27
     gate_github:
       - tox_py27
+     
+  - name: rust-unofficial/too-many-lists
+    check_github: 
+      - bonncy-run-check
```
Comment (omgjlk) [inline roles/zuul/templates/etc/zuul/config/layout.yaml]: Typo, bonnyci not bonnycy.

File: roles/zuul/templates/etc/zuul/config/layout.yaml [hunk 2]
```diff
@@ -104,3 +104,9 @@ projects:
...
+    gate_github:
+      - bonncy-run-gate
```
Comment (omgjlk) [inline roles/zuul/templates/etc/zuul/config/layout.yaml]: Same here.

**Review by omgjlk — CHANGES_REQUESTED** [review]

**Review by rattboi — DISMISSED** [review]
Looks like typos.

s/bonncy/bonnyci/

**Comment by edunham** [comment]
I'm a noob who's following your noob docs: http://bonnyci.org/lore/end_users/setup/

```
Zuul Layout Configuration

We need to add your project to the list of projects that will be sending webhook events to BonnyCI. Edit hoist/roles/zuul/templates/etc/zuul/config/layout.yaml and add an entry in the projects: list for your project. Ensure it is in alphabetical order. Here is a sample configuration:

projects:
  ...

  - name: unique_job_name
    check_github:
      - bonncy-run-check
    gate_github:
      - bonncy-run-gate

  ...
```

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/hoist/186/1488583112.29/bonnyci-run-check-multinode : FAILURE in 7m 17s

**Comment by edunham** [comment]
Commit amended. I hope bonny is cool with that.

**Review by omgjlk — APPROVED** [review]

**Comment by omgjlk** [comment]
gate_recheck

**Review by ConsoleCatzirl — APPROVED** [review]

**Pull request merged**